nextflow_schema.json

```diff
@@ -0,0 +1,192 @@
+            "type": "object",
+            "description": "Options for quast",
+            "default": "",
+            "properties": {
+                "quast_ref": {
+                    "type": "string",
+                    "default": "data/C9_S.reference.fna",
+                    "format": "file-path",
+                    "fa_icon": "fas fa-file-alt",
+                    "description": "Path to reference fasta file to base quast metrics off of"
+                }
+            }
+        },
+        "chewbbaca_options": {
+            "title": "Chewbbaca options",
+            "type": "object",
+            "description": "Options to specify the chewbbaca schema to use",
+            "default": "",
+            "properties": {
+                "schema_targets": {
+                    "type": "string",
+                    "description": "Schema targets"
+                },
+                "prepped_schema": {
+                    "type": "string",
+                    "default": "data/SeqSphere_1521_schema",
+                    "description": "Chewbbaca prepped schema",
+                    "format": "directory-path"
+                }
+            },
+            "fa_icon": "fab fa-cc-diners-club"
+        },
+        "generic_options": {
+            "title": "Generic options",
+            "type": "object",
+            "fa_icon": "fas fa-file-import",
+            "description": "Less common options for the pipeline, typically set in a config file.",
+            "help_text": "These options are common to all nf-core pipelines and allow you to customise some of the core preferences for how the pipeline runs.\n\nTypically these options would be set in a Nextflow config file loaded for all pipeline runs, such as `~/.nextflow/config`.",
+            "properties": {
+                "outdir": {
+                    "type": "string",
+                    "format": "directory-path",
+                    "description": "The output directory where the results will be saved. You have to use absolute paths to storage on Cloud infrastructure.",
+                    "fa_icon": "fas fa-folder-open",
+                    "default": "results"
+                },
+                "publish_dir_mode": {
+                    "type": "string",
+                    "default": "copy",
+                    "description": "Method used to save pipeline results to output directory.",
+                    "help_text": "The Nextflow `publishDir` option specifies which intermediate files should be saved to the output directory. This option tells the pipeline what method should be used to move these files. See [Nextflow docs](https://www.nextflow.io/docs/latest/process.html#publishdir) for details.",
+                    "fa_icon": "fas fa-copy",
+                    "enum": ["symlink", "rellink", "link", "copy", "copyNoFollow", "move"],
+                    "hidden": true
+                },
+                "max_memory": {
+                    "type": "string",
```

Comment:
I'd recommend marking these all as `"hidden": true` so they don't show up in IRIDA Next. We configure resource settings behind-the-scenes for IRIDA Next.